contracts/SaleHybridToken.sol

```diff
@@ -4,6 +4,9 @@ pragma solidity >=0.6.6;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import "./presale/AlphaPresale.sol";
```

Comment:
Use interface import instead of importing all of the SC's bytecode. This import would drastically increase deployment gas costs and should only be used for Factory pattern.

Reply:
Done